client/analysis/output.go

```diff
@@ -353,6 +380,20 @@ func printSTDOUTOutputYarnAudit(issues []types.HuskyCIVulnerability) {
 	}
 }
 
+func printSTDOUTOutputSpotBugs(issues []types.HuskyCIVulnerability) {
+	for _, issue := range issues {
+		fmt.Println()
+		fmt.Printf("[HUSKYCI][!] Language: %s\n", issue.Language)
+		fmt.Printf("[HUSKYCI][!] Tool: %s\n", issue.SecurityTool)
+		fmt.Printf("[HUSKYCI][!] Confidence: %s\n", issue.Confidence)
```

Comment:
A `fmt.Printf("[HUSKYCI][!] Severity: %s\n", issue.Severity)` is missing here! 😅